build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
 
     compile 'com.microsoft.azure:azure-keyvault:1.0.0'
     compile 'com.microsoft.azure:adal4j:1.5.0'
+    implementation 'com.google.http-client:google-http-client:1.27.0'
```

Comment:
Could we not have used an http client already in the project?

Reply:
Other client does not have straight away support for exponential back off

Reply:
Using Feign client with retryer